cfme/automate/explorer/domain.py

```diff
@@ -264,6 +278,26 @@ def update(self, updates):
             view.flash.assert_message(
                 'Edit of Automate Domain "{}" was cancelled by the user'.format(self.name))
 
+    def delete_if_exists(self):
```

Comment:
@ganeshhubale now this method cover under `BaseEntity`. I think no need of it. please check it one time.